app/views/list.scala.html

```diff
@@ -39,9 +44,9 @@
 
     <h1 id="homeTitle">@Messages("computers.list.title", currentPage.getTotalCount)</h1>
 
-    @if(flash.containsKey("success")) {
+    @request.flash.asScala().get("success").map { successFlashValue =>
```

Comment:
If there is an `implicit Http.Request` then you can use `@flash("key")` (also returns a `Optional`).

Reply:
Then I need to review a few other samples.